frontend/src/static/js/v_summary.js

```diff
@@ -413,21 +413,19 @@ window.vSummary = {
 
       const res = [];
 
-      const sinceDate = dateRounding(this.filterSinceDate, 0); // round up for the next monday
+      const nextMondayDate = dateRounding(this.filterSinceDate, 0); // round up for the next monday
       const untilDate = this.filterUntilDate;
 
-      const sinceMs = (new Date(sinceDate)).getTime();
+      const nextMondayMs = (new Date(nextMondayDate)).getTime();
       const untilMs = (new Date(untilDate)).getTime();
+      const sinceMs = new Date(this.filterSinceDate).getTime();
```

Comment:
Move `sinceMs` before `untilMs` to minimize changes.

Reply:
Made the changes.